pandas/sparse/tests/test_arithmetics.py

```diff
@@ -14,99 +14,107 @@ def _assert(self, a, b):
         tm.assert_numpy_array_equal(a, b)
 
     def _check_numeric_ops(self, a, b, a_dense, b_dense):
-        # sparse & sparse
-        self._assert((a + b).to_dense(), a_dense + b_dense)
-        self._assert((b + a).to_dense(), b_dense + a_dense)
+        with np.errstate(invalid='ignore', divide='ignore'):
+            # Unfortunately, trying to wrap the computation of each expected
```

Comment:
instead of wrapping *inside* ``_check_numeric_ops``, could rename this and create another function which calls it (or just wrap the callers, which might be a lot :<)